frontend/src/queries/nodes/DataVisualization/Components/ChartSelection.tsx

```diff
@@ -0,0 +1,44 @@
+import './ChartSelection.scss'
+
+import { LemonLabel, LemonSelect } from '@posthog/lemon-ui'
+import { useActions, useMountedLogic, useValues } from 'kea'
+
+import { dataVisualizationLogic } from '../dataVisualizationLogic'
+
+export const ChartSelection = (): JSX.Element => {
+    const logic = useMountedLogic(dataVisualizationLogic)
+    const { columns, responseLoading, selectedXIndex, selectedYIndex } = useValues(logic)
+    const { setXAxis, setYAxis } = useActions(logic)
```

Comment:
```suggestion
    const { columns, responseLoading, selectedXIndex, selectedYIndex } = useValues(dataVisualizationLogic)
    const { setXAxis, setYAxis } = useActions(dataVisualizationLogic)
```

The `useMountedLogic` hook is usually relevant only if you want some logic to mount with the component in order to start a background task, without using any actual values/actions directly. It's not really needed here.